modules/users/client/components/Monkeybox.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import keyBy from 'lodash/keyBy';
+import { useTranslation } from 'react-i18next';
+
+import Avatar from '@/modules/users/client/components/Avatar.component';
+import { userType } from '@/modules/users/client/users.prop-types';
+
+function TribesInCommon({ user, otherUser }) {
+  const { t } = useTranslation('messages');
+
+  const [tribesInCommon, setTribesInCommon] = useState([]);
+
+  useEffect(() => {
+    // otherUser will have a filled out tribe details, my user won't...
+    const byId = keyBy(
+      otherUser.member.map(membership => membership.tribe),
+      tribe => tribe._id,
+    );
+    const tribesInCommon = user.memberIds
+      .filter(id => byId[id])
+      .map(id => byId[id]);
+    setTribesInCommon(tribesInCommon);
+  }, [user, otherUser]);
+
+  if (tribesInCommon.length === 0) return null;
+
+  return (
+    <div className="monkeybox-section">
+      <div className="tribes-common">
+        <h4>{t('Tribes in common')}</h4>
+        <ul className="list-inline">
+          {tribesInCommon.map(tribe => (
+            <li key={tribe._id}>
+              <a className="tribe-link" href={`/tribes/${tribe.slug}`}>
+                {tribe.label}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+    </div>
+  );
+}
+
+TribesInCommon.propTypes = {
+  user: userType.isRequired,
+  otherUser: userType.isRequired,
+};
+
+export default function Monkeybox({ user, otherUser }) {
+  return (
+    <div className="monkeybox panel panel-default">
+      <div className="panel-body">
+        <Avatar user={user} size={64} />
+        <h3>
+          <a>{user.displayName}</a>
+        </h3>
+        <TribesInCommon user={user} otherUser={otherUser} />
+        {user.languages.length > 0 && (
+          <div className="monkeybox-section">
```

Comment:
For some reason when in my test `/api/users/[USERNAME]` response has `languages: []`, I'm still getting `Warning: Each child in a list should have a unique "key" prop.
` warning for this line and I don't understand why. 🤔  This section should never render...

Reply:
The section is actually rendering wrong profile — it shows up my own profile, while it should show the other user's profile:

<img width="548" alt="image" src="https://user-images.githubusercontent.com/87168/78456024-25dd4f00-76aa-11ea-9bd5-e6062bbb8605.png">

Reply:
Good catch for the wrong user! Fixed in https://github.com/Trustroots/trustroots/pull/1173/commits/70ee62cf94d3b378be9dca831185718c44437dad

I can't recreate the unique "key" issue... and it doesn't make sense as you pointed out. I'm wondering if something else caused that issue. Maybe a user can have multiple non-unique langauges in the seed data? I'll leave it out for now...